_config.yml

```diff
@@ -35,6 +35,41 @@ author:
   photo                     : "uploads/SSE_LOGO.png"
   photo2x                   : "uploads/SSE_LOGO.png"
 
+eboard:
+  sabrina:
+    fullname: Sabrina Stangler™️
+    email: stanglersm@msoe.edu
+    linkedin: linkedin.com/in/sabrinastangler/
+    twitter: sabrinastangler
+  seth:
+    fullname: Seth Fenske
+    email: fenskesd@msoe.edu
+    phone: (262) 977 - 4026
```

Comment:
Consistency.  Why different phone number formatting on this line?

Reply:
Unfocused coding